src/spectrogramplot.cpp

```diff
@@ -169,6 +169,8 @@ void SpectrogramPlot::paintAnnotations(QPainter &painter, QRect &rect, range_t<s
     painter.setPen(pen);
     QFontMetrics fm(painter.font());
 
+    annotationLocations.clear();
+    
```

Comment:
The command line diff says that there are trailing white spaces

Reply:
Sorry. I didn't notice. I have removed the whitespace.